app/views/Project/EntryEdit/index.tsx

```diff
@@ -424,10 +424,12 @@ function EntryEdit(props: Props) {
                                 ...entry,
                                 deleted: undefined,
                                 stale: undefined,
-                                attributes: entry.attributes?.map((attribute) => ({
-                                    ...attribute,
-                                    widgetType: undefined,
-                                })),
+                                attributes: entry.attributes
+                                    ?.filter((attribute) => isDefined(attribute.data))
+                                    .map((attribute) => ({
+                                        ...attribute,
+                                        widgetType: undefined,
+                                    })),
```

Comment:
Please verify this logic @thenav56 @AdityaKhatri @samshara

Reply:
Looks good. We are deleting attributes without data right?